src/runtime/hexagon_dma_rt.h

```diff
@@ -0,0 +1,179 @@
+/*!
+ * This file contains the prototypes of the Halide hexagaon DMA transfer runtime functions
+ * These functions will use the parameters passed by the user through Halide pipeline and populate
+ * the Hexagon dma context structure, the details in this structure will ultimately be used by hexagon
+ * dma device interface for invoking actual DMA transfer
+ */
+
+#ifndef _DMA_HALIDE_RT_H_
+#define _DMA_HALIDE_RT_H_
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*!
```

Comment:
These public API functions should use doxygen style comments consistent with other public Halide APIs.